registry-manager-service-api/src/main/java/de/adorsys/registry/manager/service/AspspService.java

```diff
@@ -37,6 +37,14 @@
 
     AspspBO save(AspspBO aspsp);
 
+    /**
+     * Checks if an incoming Aspsp record is already represented in the database
+     *
+     * @param aspsp to check
+     * @return result if there are any duplicates
+     */
+    AspspBO checkNewAspsp(AspspBO aspsp);
```

Comment:
Seems to me this method has wrong signature. if the name started from "check" we are expecting boolean result but an object

Reply:
renamed